### PR TITLE
implemented #2314: SqlUtil: add support for custom column operators

### PR DESCRIPTION
**Pull request by pvanek**

### CODE REVIEW

File: qlib/SqlUtil.qm [hunk 1]
```diff
@@ -14612,10 +14614,75 @@ string sql = table.getAlignSqlString(table2);
         }
 
         #! returns the column operator map for this object
-        /** override in subclasses to return driver-specific options
+        /** subclasses has to implement getColumnOperatorMapImpl() to return driver-specific options
```
Comment (davidnich) [inline qlib/SqlUtil.qm]: please update to `have to`

**Review by davidnich — CHANGES_REQUESTED** [review]
please add a note in doxygen/lang/900_release_notes.dox.tmpl

also the SqlUtil modules should get a point update and notes

**Comment by pvanek** [comment]
@davidnich relnotes added

**Review by davidnich — CHANGES_REQUESTED** [review]
@pvanek please update SqlUtil to v1.4.1 and move the relnote there - technically the other modules should also get version bumps and relnotes as well

**Comment by pvanek** [comment]
@davidnich vesrion bump done

**Pull request merged**